modules/ui.js

```diff
@@ -0,0 +1,31 @@
+import LocalStore from './localStore.js';
+
+const bookDetail = document.querySelector('.books-detail');
+let newId = 0;
+
+class UI {
+  static showBooks() {
```

Comment:
- [ ]  Kindly make use of the ES6 arrow function for your approach to be of the latest and it would help you in React module. Apply this to others too, please.